content/en/security/threat_intel.md

```diff
@@ -0,0 +1,88 @@
+
+**Application Packages: Malware Risk in Distribution**
+
+Unlike immutable file hashes, application packages can vary in content and security, even under the same version number. Malicious actors may upload harmful packages mimicking legitimate ones, or they might compromise existing packages by introducing malware. The lifecycle of malicious packages is frequently long-lived, but not immutable.
+
+**Domains: Temporary Signatures**
+
+Unlike file hashes, domains identified as malicious are subject to change. They may undergo processes such as remediation, reassignment, or repurposing by various entities. While the lifecycle of malicious or suspicious domains is somewhat prolonged compared to IP addresses, it remains temporary and variable.
+
+**IP Addresses: Dynamic and Transient** 
+
+IP addresses represent the most volatile element in threat intelligence, often changing reputations within a 24-hour cycle. Given their dynamic nature, particularly in residential and mobile networks where multiple hosts may be involved, it's crucial to regularly reassess their status. Not all hosts connected to a low-reputation IP address are inherently malicious, underscoring the need for correlation.
+
+## Best Practices in Threat Intelligence
+
+With threat intelligence, reputation is key, but it must be weighed alongside other evidence. Relying solely on IP and domain intelligence for blocking traffic is not recommended, with few exceptions. A balanced, evidence-based approach is essential.
+
+Threat intelligence used in Detection Rules should reference the Datadog keys such as category and intent. Other keys should not be used.
+
+## Transparency in Threat Intelligence 
+Datadog ensures transparency by providing external links to external threat intelligence sources associated with a detection. Threat intelligence curated by Datadog is ingested into the Datadog platform for enrichment and detection. Datadog does not send customer data to threat intelligence sources.
+
+The detections and enrichments are accessible in the UI and event JSON.
+
+## Threat Intelligence Facets
+Sources, categories, and intents are available as facets and filters on relevant product Explorers. 
+
+### Threat Intelligence Sources
+
+| Source | Category | Source Use Cases | Primary Products | 
+|--------|------------|-----------|------------------|
+| Datadog Threat Research| scanners, exploits | Honeypots focused on software specific threats | ASM and CWS |
+| [Spur](https://spur.us/) | residential_proxy | Proxies associated credential stuffing and fruad | ASM and Cloud SIEM |
+| [Spur](https://spur.us/) | malware_proxy | Proxies associated with malware command and control | Cloud SIEM |
```

Comment:
A bit odd to see Spur mentioned twice here. This looks like we're more explaining the categories (and we call out the categories twice in the subsequent category section also).

Can we merge the Spur rows into one?